qemu/tests/block_copy.py

```diff
@@ -46,6 +46,10 @@ def __init__(self, test, params, env, tag):
         self.data_dir = data_dir.get_data_dir()
         self.device = self.get_device()
         self.image_file = self.get_image_file()
+        self.params["monitor_type"] = "qmp"
```

Comment:
@xutian need to modify several cfg files this way, consider of the influence, I prefer to handle this monitor issue later, I will create a new bz for it. Could you help merge current modification first? Thanks.